modules/graceful-restart.adoc

```diff
@@ -64,7 +64,31 @@ $ oc describe csr <csr_name> <1>
 $ oc adm certificate approve <csr_name>
 ----
 
-. After the control plane nodes are ready, verify that all worker nodes are ready.
+. After the control plane nodes are ready, mark all the nodes in the cluster as schedulable. You can do this from your cloud provider's web console, or by running the following loop:
++
+[source,terminal]
+----
+for node in $(oc get nodes -o jsonpath='{.items[*].metadata.name}'); do echo ${node} ; oc adm uncordon ${node} ; done
+----
++
+.Example output
```

Comment:
Not sure we need this example here.

Reply:
I agree, we might want to remove the example output if it is unnecessary. I was just aligning the [shutdown step](https://docs.openshift.com/container-platform/4.14/backup_and_restore/graceful-cluster-shutdown.html#graceful-shutdown-cluster) to this equivalent restart step.